### PR TITLE
Single blob shape + fix blob alignment

### LINKED ISSUE #1896: The bounding box for the blobs should not contain empty spaces.
## Bug Description

The bounding box for the blobs should not contain empty spaces.

## Steps to Reproduce

1. Add a blob shape
2. click on it / make it the focus

## Additional Context

- Plugin Version:
- Operating System:
- Browser:

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

### PR DESCRIPTION
**Pull request by wassgha**
## Summary

Temporarily removes the 8 different blob shapes and replaces them with a single shape in preparation for #686 

## Relevant Technical Choices

Added a migration script that switches all blobs to the single blob object.

## User-facing changes

❗❗ **This alters the appearance of stories that used the old blobs in two ways: Stories that used `blob-1` will now see a bigger blob since the path was changed to fill empty space and stories that used other blobs will be replaced by `blob-1`** 

## Testing Instructions

1. Drag a blob element into the canvas
2. Notice how the outline, drop target and frame align to the blob's shape (no longer showing empty space around it)
---

Fixes #1896

### CODE REVIEW

**Review by dvoytenko — APPROVED** [review]

File: assets/src/edit-story/migration/migrations/v0024_blobsToSingleBlob.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+function reducePage({ elements, ...rest }) {
+  return {
+    elements: elements.map(updateElement),
+    ...rest,
+  };
+}
+
+function updateElement(element) {
+  if (element?.mask?.type && element?.mask?.type.startsWith('blob-')) {
+    element.mask.type = 'blob';
```
Comment (barklund) [inline assets/src/edit-story/migration/migrations/v0024_blobsToSingleBlob.js:33]: Are the migrations okay with editing the existing objects?

**Review by barklund — APPROVED** [review]
So, UX and Product is okay with this, right? Otherwise it looks fine (except my carefully crafted blob images are now rather different).

**Comment by barklund** [comment]
Please make sure to add full QA instructions to the underlying ticket, #1896.

**Comment by wassgha** [comment]
@pbakaus or @samitron7 to confirm that this temporary change is ok

**Comment by pbakaus** [comment]
works for me! Thanks

**Pull request merged**